databases/cloud-pubsub/deployment/pubsub-with-workload-identity.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_deployment_pubsub_with_workflow_identity_deployment_pubsub]
+# [START container_pubsub_secret_deployment]
```

Comment:
This probably needs to have a different name; likely `[START container_pubsub_workload_identity_deployment]` or similar. (Looks like it was copied from the pubsub-with-secret.yaml file?)

Also below (line 34)

Reply:
Thank you, done.